examples/python/video_api.ipynb

```diff
@@ -134,19 +134,32 @@
    "metadata": {},
    "source": [
     "Here we can see that video has two streams - a video and an audio stream. \n",
-    "\n",
-    "Let's read all the frames from the video stream."
+    "Currently available stream types include ``['video', 'audio']``.\n",
+    "Each descriptor consists of two parts: stream type (e.g. 'video') and\n",
+    "a unique stream id (which are determined by video encoding).\n",
+    "In this way, if the video contaner contains multiple\n",
+    "streams of the same type, users can acces the one they want.\n",
+    "If only stream type is passed, the decoder auto-detects first stream\n",
+    "of that type and returns it.\n",
+    "\n",
+    "Let's read all the frames from the video stream.\n",
+    "By default, the return value of `next(video_reader)` is a dict (**this might change**, but _should_ be backwards compatible).\n",
```

Comment:
let's remove the `**this might change, but should be ...**. We announce in the release notes that the video reading API is beta and might change, so there are more things that might change than just the return type.
```suggestion
    "The return value of `next(video_reader)` is a dict.\n",
```